### PR TITLE
Create book repository

### PR DESCRIPTION
**Pull request by juanwongramirez**
## Summary
- Create BookRepository in package repositories

## Trello Card

https://trello.com/c/E7tPNVPO

### CODE REVIEW

File: src/main/java/wolox/training/repositories/BookRepository.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package wolox.training.repositories;
+
+import java.util.Optional;
+import org.springframework.data.repository.CrudRepository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends CrudRepository<Book, Long> {
+
+    public Optional<Book> findFirstByAuthorIgnoreCase(String author);
```
Comment (Jimenarosello) [inline src/main/java/wolox/training/repositories/BookRepository.java:11]: Nice use of Optional and CrudRepository! 👏 😄

**Review by Jimenarosello — APPROVED** [review]
Nice job!! 

approve

**Review by jeymarsonwolox — APPROVED** [review]
Excellent!!

**Review by Jimenarosello — APPROVED** [review]
🚀 😄  Nice!

**Pull request merged**